### PR TITLE
Use latest patch of DefaultKubernetesRelease as default k8s version

### PR DESCRIPTION
**Pull request by CecileRobertMichon**
**What this PR does / why we need it**: Update k8s default version from 1.8.9 to 1.8.11

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**If applicable**:
- [ ] documentation
- [ ] unit tests
- [ ] tested backward compatibility (ie. deploy with previous version, upgrade with this branch)

**Release note**:

```release-note
```

### CODE REVIEW

**Comment by jackfrancis** [comment]
Should we instead prefer a release channel (i.e., 1.8) and then just make the default version the latest patch release in that channel?

**Comment by CecileRobertMichon** [comment]
Yes I think that makes sense. The only reason I see that would prevent us from doing that is baking newly released k8s minor versions in acs-engine before we make them default. For example, when k8s 1.8.12 comes out, we add support for it right away but we only promote it to the default version after it has been passing our 1.8 tests for a while. What do you think?

**Comment by jackfrancis** [comment]
I think we defer to k8s to bake basic stability into the most current release. But open to doing anything reasonable that is automateable.

**Comment by CecileRobertMichon** [comment]
Ok agreed, I'll modify this PR to implement a default release instead of a default version

File: pkg/api/common/versions.go [hunk 1]
```diff
@@ -214,9 +219,9 @@ func GetSupportedVersions(orchType string, hasWindows bool) (versions []string,
 	switch orchType {
 	case Kubernetes:
 		if hasWindows {
-			return GetAllSupportedKubernetesVersionsWindows(), string(KubernetesDefaultVersion)
+			return GetAllSupportedKubernetesVersionsWindows(), GetDefaultKubernetesVersion()
 		}
-		return GetAllSupportedKubernetesVersions(), string(KubernetesDefaultVersion)
```
Comment (CecileRobertMichon) [inline pkg/api/common/versions.go]: Any reason why a string conversion was necessary here? I removed it but let me know if I'm missing something @jackfrancis

Comment (jackfrancis) [reply pkg/api/common/versions.go]: not a sane one that I can think of, nope!

**Pull request merged**